### DISCUSSION TITLE
Specifying path in links for disambiguation?

### DISCUSSION THREAD

**Discussion by Anrock** [nvim-neorg/neorg#727] [Syntax] [answered]
So for example I have multiple heading with the same title, like
```
* Foo
** Implementation

* Bar
** Implementation

* Additional info
  See {** Implementation}
```

Right now it seems that the link will lead to first encountered header with matching title.

But what if I want the link to lead to `Bar` `Implementation` header? Is it possible to disambiguate a link like `{* Bar ** Implementation}` or something like that? Or is there a better way to organize a document?

**Comment by vhyrro** [comment, marked as answer]
The current parser does not support this properly (because it was added at the very end of development), however as per the spec it is possible to use the `:` char with spaces around it to do what you want:

```
{* Level 1 : ** Level 2 Heading}
```

This also applies to any nested object, for example if you want a footnote within a definition:
```
{$ Definition : ^ Footnote}
```

A second gen parser is in the works and will support this natively.

**Reply by Anrock** [reply]
Just noticed that following `{* Level 1 : ** Level 2 Heading}` actually takes me to Level 1 header instead of Level 2. Is this a new bug or a consequence of "current parser does not support this properly"?
